### DISCUSSION TITLE
Looking for another exporter

### DISCUSSION THREAD

**Discussion by baruchiro** [brafdlog/caspion#255] [Ideas]
Hi, I'm trying to share the app with end-users, but I can't "sell" them since it is still not a complete solution.

If you don't have a YNAB account, you need to export your data to kind of "raw" data- JSON, CSV, Google Sheets and start to analyze it yourself.

I'm looking for a free app to export to talk with potential users and suggest they try that app.

**Comment by baruchiro** [comment]
@brafdlog @MordRot @giladtaase @Arielgordon123 @baaraak

**Comment by baruchiro** [comment]
[expense-tracker](https://www.deborahho.com/productivity/expense-tracker)

It actually a template for Google Spreadsheet and Google Form, I don't think it so much better than DataStudio.

**Reply by brafdlog** [reply]
Benefit here is that it is free and doesn't require any other app

**Reply by baruchiro** [reply]
Don't know, it looks complicated. I will use Google Data Studio because I already know how to use it, and I'm just looking for a solution to give to the users (and I expect them to choose their solution after using our app for a while).

**Comment by baruchiro** [comment]
[spendee](https://www.spendee.com/)

Support Israeli banks import!

- [ ] Manual import (no API)
- [x] Free version

**Comment by baruchiro** [comment]
[dollarbird](https://dollarbird.co/)

- [ ] No free version

**Comment by baruchiro** [comment]
[pocketsmith](https://www.pocketsmith.com/)

**Comment by baruchiro** [comment]
[toshl](https://toshl.com/)

- [ ] Limited free version

**Comment by baruchiro** [comment]
[moneylover](https://moneylover.me/)

- [ ] Can't understand the pricing yet
- [ ] [API](https://oapi.moneylover.me/docs/)

**Comment by baruchiro** [comment]
[travel-spend](https://travel-spend.com/)

**Comment by baruchiro** [comment]
[Google Data Studio Report Template](https://developers.google.com/datastudio/connector/report-templates)

- [x] Free (I think)
- [x] Unlimited
- [x] Already implemented the Google Sheets exporter

**Comment by baruchiro** [comment]
[creditCardAnalysis](https://github.com/raziele/creditCardAnalysis)

- [ ] Requested to support our format: https://github.com/raziele/creditCardAnalysis/issues/10
